modules/cms/models/MaintenanceSetting.php

```diff
@@ -3,6 +3,8 @@
 use Model;
 use Cms\Classes\Page;
 use Cms\Classes\Theme;
+use Illuminate\Support\Arr;
```

Comment:
Doesn't October have it's own version?

Reply:
Yep:
```
<?php namespace Winter\Storm\Support;

use Illuminate\Support\Arr as ArrHelper;

/**
 * Array helper
 *
 * @author Alexey Bobkov, Samuel Georges
 */
class Arr extends ArrHelper
```

Reply:
Yeah my bad, it got auto completed and I didn't check the import. Updated to Winter's version :)